src/mpc-hc/MainFrm.cpp

```diff
@@ -16472,19 +16472,13 @@ bool CMainFrame::LoadSubtitle(CString fn, SubtitleInput* pSubInput /*= nullptr*/
             WIN32_FIND_DATA fd = {0};
             HANDLE hFind;
             
-            hFind = FindFirstFile(path + L"*.ttf", &fd);
+            hFind = FindFirstFile(path + L"*.*", &fd);

```

Comment:
`*.?t?`

Reply:
Also .otf

Reply:
> `*.?t?`

Yeah that should work. There may be other font types to support later.

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-addfontresourceexa

Other possible extensions.  I don't think we'd use any.

Reply:
No. Current three should cover everything.